docs/ToolsAndExamples/SDKExamples/OnchainMetadata/objectives.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 1
+---
+
```

Comment:
I'd put in a table of contents like alchemy here: https://docs.alchemy.com/docs/how-to-add-royalties-to-an-erc-20-token